nexus/src/app/db_metadata.rs

```diff
@@ -0,0 +1,7 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Access to DB-internal metadata.
+
+impl super::Nexus
```

Comment:
Mistaken file?

Reply:
Removed, thought I needed it, forgot about it.